go.mod

```diff
@@ -73,3 +74,5 @@ replace (
 	github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
 	github.com/dop251/goja_nodejs => github.com/dop251/goja_nodejs v0.0.0-20171011081505-adff31b136e6
 )
+
+replace github.com/elastic/beats/v7 v7.0.0-alpha2.0.20220627193119-7b5ef7832848 => /Users/leehinman/src/beats
```

Comment:
This is necessary to use PR elastic/beats#32258, will remove this & take this PR out of draft once 32258 is merged